### DISCUSSION TITLE
Uninstalling Cypress

### DISCUSSION THREAD

**Discussion by mcivora** [cypress-io/cypress#26283] [Questions and Help]
I'm looking to uninstall Cypress and make a clean install. 

I thought I'd done this but can still see a Cypress folder with e2e, Fixtures and Support folders below it.

How do I uninstall properly?

**Comment by ghost** [comment]
https://docs.npmjs.com/uninstalling-packages-and-dependencies

**Reply by MikeMcC399** [reply]
That only covers uninstalling the npm module `cypress`. It does not remove "Cypress folder with e2e, Fixtures and Support" that the original question referred to.

**Comment by MikeMcC399** [comment]
@mcivora

This is an old question, so I guess you will have figured it out by now.

Just for the record, uninstalling Cypress (with `npm uninstall cypress`, `pnpm remove cypress`, `yarn remove cypress`) does not delete the files and folders it may have created going through the `cypress open` sequence and you would need to remove these manually in order to do a clean install.

The [names of the files and folders](https://docs.cypress.io/guides/core-concepts/writing-and-organizing-tests#Folder-structure) depends on whether you are using JavaScript or TypeScript and whether Cypress has been set up for E2E testing, Component Testing or both. For example, JavaScript / E2E would create:

```text
/cypress.config.js
/cypress/fixtures/example.json
/cypress/support/commands.js
/cypress/support/e2e.js
```

Cypress also caches data, so to completely remove Cypress you would also need to remove this as well. That is not usually necessary if you just want to run a clean install.
